Line List: Display medium information from metadata (#1626)
spacetelescope/jdaviz: CHANGES.rst

- User-friendly API access to plugins. [#1401]

> - Line Lists show which medium the catalog wavelengths were measured in,
>   in accordance to the metadata entry [#1626]

Reviewer: ```suggestion
- Line Lists show whether the catalog wavelengths were measured in air or vacuum. [#1626]
```

I'm going off of @PatrickOgle's statement that "medium" without context might not be informative.

Reviewer: Although now that I read it again I guess the context is there 😆
